packages/webchat/src/components/ContextMenu.tsx

```diff
@@ -1,12 +1,17 @@
 import { ContextMenu, Menu, MenuItem } from '@blueprintjs/core'
 import React from 'react'
 
-function MessageMenu(props) {
+function MessageMenu(props: any) {
   return (
     <Menu>
-      {props.customActions.map(action => {
+      {props.customActions.map((action: any) => {
         return (
-          <MenuItem key={action.id} text={action.label} onClick={action.onClick.bind(this, action.id, props.element)} />
+          <MenuItem
+            key={action.id}
+            text={action.label}
+            // TODO: undefined is supposed to be "this". Does that make sense??
+            onClick={action.onClick.bind(undefined, action.id, props.element)}
```

Comment:
Before we had `onClick.bind(this, ...)` but typescript tells us this makes no sense and produces a typing error.